src/static/riot/competitions/form/_phases.tag

```diff
@@ -46,8 +46,8 @@
         <div class="content">
             <div class="ui top pointing secondary menu">
                 <a class="active item" data-tab="phase_details">Phase details</a>
-                <a class="item" data-tab="phase_datasets">Datasets</a>
-                <a class="item" data-tab="phase_task">Tasks</a>
+                <a class="item" data-tab="phase_datasets" if="{ current_phase_style == 'dataset' }">Datasets</a>
+                <a class="item" data-tab="phase_task" if="{ current_phase_style == 'task' }">Tasks</a>
```

Comment:
I think these need to be show instead of if? or something else needs to get tweaked so the tabs actually work. Can probably handle as a part of the competition edit task

Reply:
Unless it broke in the mean time, was working -- need to clear cache maybe?

Reply:
The menu item changes, I just can't actually change to the tab.

Reply:
got it, you're right, `show` works